exercises/practice/sieve/.meta/Example.cs

```diff
@@ -11,13 +11,15 @@ public static int[] Primes(int limit)
 
     private static int[] InitializePrimes(int limit)
     {
-        if (limit < 2)
+        if (limit < 0)
         {
             throw new ArgumentOutOfRangeException(nameof(limit));
         }
 
-        var candidates = new Queue<int>(Enumerable.Range(2, limit - 1));
         var primes = new List<int>();
+        if (limit == 0 || limit == 1) return primes.ToArray();
```

Comment:
```suggestion
        if (limit == 0 || limit == 1) return Array.Empty<int>();
        var primes = new List<int>();
```

Reply:
Maybe change to the above? If you disagree, I'm fine with merging _as is).

Reply:
omg I was trying to do that but didn't realise why it wasn't letting me, I was missing the <int> ! :( thanks